nncf/torch/graph/operator_metatypes.py

```diff
@@ -1038,12 +1038,7 @@ def get_operator_metatypes() -> List[Type[OperatorMetatype]]:
     PTModuleConv1dMetatype,
     PTModuleConv2dMetatype,
     PTModuleConv3dMetatype,
-    PTDepthwiseConv1dSubtype,
-    PTDepthwiseConv2dSubtype,
-    PTDepthwiseConv3dSubtype,
-    PTModuleConvTranspose1dMetatype,
```

Comment:
It looks like `ConvTranspose` operations have similar semantic as `Convolution`. From BC perspective, what is strategy for it? Do you have a list of models where `ConvTranspose` operations are used?

Useful link: https://pytorch.org/docs/stable/generated/torch.nn.ConvTranspose2d.html

Reply:
Since we have only conformance models scope for the PyTorch backend, the removal of the **ConvTranspose** doesn't affect the accuracy for any model here. So, we can decide that the models from the conformance scope don't contain **ConvTranspose** or this layer didn't work anyway, and was added with no reason and testing.
@AlexanderDokuchaev, could you please clarify?

Reply:
What i should clarify? I have no list of used modules in test.
`ConvTranspose` was added as operation with bias...
But as i remember transpose convolution in OV has type `ConvolutionBackpropData`, that does not exist in OPERATIONS_WITH_BIAS for OV backend.

So, if transposed layers should not be supported it `ConvTranspose` can be removed.  In otherwise need to check that some model using `ConvTranspose` and add this kind of model to test scope if need.

And looks like `OPERATIONS_WITH_BIAS` is not correct name to current use, may be it should be `metatypes_for_fast_bias_correction`.

Reply:
> What i should clarify?

Need to clarify why you've added these layers to the **OPERATORS_WITH_BIAS_METATYPES** list. **ConvolutionBackpropData** wasn't added to the OV backend for a reason: it possibly causes accuracy degradation.

Reply:
I hope, the comment will be resolved in 106469.